students/eowyn/session02/print_grid.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+
+''' Construct grid that is composed of ndim boxes in 
+    either direction, and each box is composed of ndash 
+    lines both horizontally and vertically '''
+
+### First method uses global vars and functions for lines and verts
+### then a caller function to print it all
+
+# define global varaibles
```

Comment:
In general, global variables are considered "bad form" -- but this is a fine use case --essentially constants.

It is considerd good style to use ALL CAPS for contants, so you may want to use names like:

PLUS = "+"
MINUS = "-"

and so on. But no biggie.